plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -177,6 +177,8 @@ CODESTARTcreateWrkrInstance
 	pWrkrData->curlCheckConnHandle = NULL;
 	pWrkrData->serverIndex = 0;
 	pWrkrData->restURL = NULL;
+    pWrkrData->reply = NULL; /* initial init of pointer because of checkConn can be called before curlPost */
```

Comment:
The comment is wrong - sorry. This is not possible. However, in the longer term this is the right spot to do it. 

Will you remove the misleading comment or shall I do after merge? If it is left in, you really confuse people working on the action engine.

Reply:
Sorry, I do not know your coding/release process. (And my English is not very good(( )
Please add any changes that required.

Reply:
yeap. I understand your idea.
Yes you can remove it. I have added it just to detalise the changes.

Reply:
removed